src/inline-text/link/link.module.css

```diff
@@ -1,11 +1,8 @@
 .link {
   color: var(--figma-color-text-brand);
   pointer-events: all;
-  text-decoration: none;
```

Comment:
this library is a fork from https://yuanqing.github.io/create-figma-plugin/ui/ to be usable in backlight, it's not meant at having customization not pushed upstream, rebasing will become unmanageable otherwise.

We can achieve the same customization outside with `<Link style={{ textDecoration: 'underline' }}` ...` AFAICT

Reply:
Aah okay gotcha